examples/with-aws-amplify-typescript/pages/todo/[id].tsx

```diff
@@ -16,21 +15,40 @@ const TodoPage = (props: { todo: GetTodoQuery['getTodo'] }) => {
   )
 }
 
-TodoPage.getInitialProps = async (context) => {
-  const { id } = context.query
+export const getStaticPaths = async () => {
+  let result = (await API.graphql(
+    graphqlOperation(getTodoList, { id: 'global' })
+  )) as { data: GetTodoListQuery; errors: any[] }
+  if (result.errors) {
+    console.log('Failed to fetch todolist. ', result.errors)
+    throw new Error(result.errors[0].message)
+  }
+  if (result.data.getTodoList !== null) {
+    const paths = result.data.getTodoList.todos.items.map(({ id }) => ({
+      params: { id },
+    }))
+    return { paths, fallback: false }
+  }
```

Comment:
Let's remove this conditional to always return something, it doesn't matter if it may throw without it, as it would happen at build time, which is okay.

```suggestion
  const paths = result.data.getTodoList.todos.items.map(({ id }) => ({
    params: { id },
  }))
  return { paths, fallback: false }
```